.travis.yml

```diff
@@ -6,6 +6,10 @@ cache: 'yarn'
 script:
   - yarn test
   - yarn build
+before_install:
```

Comment:
Is this PR introducing linting or fixing the travis script?

Reply:
Yep, you are right, I was fixing the travis script in the wrong branch